### ISSUE TITLE
FileNotFoundError doesn't exist in python 2

### ISSUE THREAD

**Issue by raylu** [LearnProgramming/learntris#42]
ccca3d6ebb6b35d3836b9580b8a4da93fe9c4be0

```
Traceback (most recent call last):
  File "testris.py", line 184, in <module>
    main()
  File "testris.py", line 155, in main
    except FileNotFoundError as e:
NameError: global name 'FileNotFoundError' is not defined
```

**Comment by tangentstorm** [comment]
Well, that's annoying. I hate adding version compatibility cruft, but python 3 actually throws that exception, and it's nice to trap it explicitly, so I want to check for it. I guess we can just fall back to `IOError`. I'll take a look. Thanks.

**Comment by tangentstorm** [comment]
Fixed in 44719c9a.
